MultiplatformDemoWithSync/shared/src/commonMain/kotlin/io/realm/kotlin/demo/util/Constants.kt

```diff
@@ -19,6 +19,7 @@ package io.realm.kotlin.demo.util
  * Replace with your own App's credentials to enable Sync.
  * To setup the Realm Sync App on MongoDB Atlas follow the steps here
  * https://docs.mongodb.com/realm/tutorial/realm-app/ or watch the video tutorial https://www.youtube.com/watch?v=lqo0Yf7lnyg
+ * https://www.mongodb.com/docs/atlas/tutorial/create-atlas-account/ or watch the video tutorial https://www.youtube.com/watch?v=lqo0Yf7lnyg
```

Comment:
Was this supposed to replace the link above? If not I would probably reword this into a list:

```
To setup the Realm Sync App on MongoDB Atlas follow the steps here:
- link 1
- link 2
```

Reply:
The intention was to replace the link yes!